### PR TITLE
Spelling

### PR DESCRIPTION
**Pull request by jsoref**
### Description

This PR corrects misspellings identified by the [check-spelling action](https://github.com/marketplace/actions/check-spelling).

The misspellings have been reported at https://github.com/jsoref/gqlalchemy/actions/runs/5270875048#summary-14266846969

The action reports that the changes in this PR would make it happy: https://github.com/jsoref/gqlalchemy/actions/runs/5270875251#summary-14266847585

### Pull request type

Please delete options that are not relevant.

- [x] Bugfix
- [x] Code style update (formatting, renaming)
- [ ] Refactoring with functional or API changes
- [ ] Refactoring without functional or API changes
- [ ] Documentation content changes
- [ ] Other (please describe):

### Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes

######################################

### Reviewer checklist (the reviewer checks this part)
- [ ] Core feature implementation
- [ ] Tests
- [ ] Code documentation
- [ ] Documentation on [memgraph/docs](https://github.com/memgraph/docs)

######################################

### CODE REVIEW

File: docs/reference/gqlalchemy/transformations/translators/nx_translator.md [hunk 1]
```diff
@@ -59,6 +59,6 @@ Generates Cypher queries and inserts data into Memgraph in parallel.
 def get_instance()
 ```
 
-Creates Networx instance of the graph from the data residing inside Memgraph. Since Networkx doesn&#x27;t support labels in a way Memgraph does, labels
+Creates Networkx instance of the graph from the data residing inside Memgraph. Since Networkx doesn&#x27;t support labels in a way Memgraph does, labels
```
Comment (jsoref) [inline docs/reference/gqlalchemy/transformations/translators/nx_translator.md:62]: brand

File: gqlalchemy/graph_algorithms/query_modules.py [hunk 2]
```diff
@@ -25,10 +25,10 @@
 QM_FIELD_PATH = "path"
 QM_FIELD_SIGNATURE = "signature"
 
-LEFT_PARANTHESES = "("
-RIGHT_PARANTHESES = ")"
+LEFT_PARENTHESIS = "("
+RIGHT_PARENTHESIS = ")"
```
Comment (jsoref) [inline gqlalchemy/graph_algorithms/query_modules.py:29]: parentheses are plural, parenthesis is singular...

File: tests/integration/test_constraints.py [hunk 3]
```diff
@@ -166,11 +166,11 @@ def test_ensure_constraints(memgraph):
...
             "NodeThree",
             (
-                "attribtue1",
-                "attribtue2",
+                "attribute1",
+                "attribute2",
             ),
         ),
-        MemgraphConstraintUnique("NodeThree", ("attribtue3",)),
+        MemgraphConstraintUnique("NodeThree", ("attribute3",)),
```
Comment (jsoref) [inline tests/integration/test_constraints.py:173]: Sometimes typos are intentional in tests. I saw no evidence of that here.

File: tests/transformations/translators/test_dgl_transformations.py [hunk 4]
```diff
@@ -834,7 +834,7 @@ def test_dgl_import_homogeneous(memgraph):
         ]
     )
     graph = dgl.DGLGraph((src, dst))
-    # Initialize translator and insert into the MemgrapA
+    # Initialize translator and insert into the Memgraph
```
Comment (jsoref) [inline tests/transformations/translators/test_dgl_transformations.py:837]: ??

**Review by jsoref — COMMENTED** [review]
Most corrections automatically suggested by Google Sheets. All fault mine.

**Comment by katarinasupe** [comment]
Thank you @jsoref for creating this PR. We will review it for the next release and include it! :D

**Review by antepusic — APPROVED** [review]

**Comment by katarinasupe** [comment]
Hi @jsoref, I saw you created similar PRs on other Memgraph repos. Since I am working in Developer Experience for Memgraph, I was wondering if you are using Memgraph for some of your projects. I saw on your GH that you sometimes make a contribution because you're considering using a project 😄 P.S. The community mostly hangs out at Discord, [so feel free to join](https://www.discord.gg/memgraph).

File: docs/reference/gqlalchemy/transformations/translators/nx_translator.md (hunk 1 above)
Comment (katarinasupe) [reply docs/reference/gqlalchemy/transformations/translators/nx_translator.md:62]: I think NetworkX is the most correct then?

File: gqlalchemy/transformations/translators/nx_translator.py [hunk 5]
```diff
@@ -254,7 +254,7 @@ def _check_for_index_hint(
             )
 
     def get_instance(self):
-        """Creates Networx instance of the graph from the data residing inside Memgraph. Since Networkx doesn't support labels in a way Memgraph does, labels
+        """Creates Networkx instance of the graph from the data residing inside Memgraph. Since Networkx doesn't support labels in a way Memgraph does, labels
```
Comment (katarinasupe) [inline gqlalchemy/transformations/translators/nx_translator.py:257]: Again, NetworkX?

File: docs/reference/gqlalchemy/transformations/translators/nx_translator.md (hunk 1 above)
Comment (katarinasupe) [reply docs/reference/gqlalchemy/transformations/translators/nx_translator.md:62]: I think NetworkX is the most correct then?

File: tests/integration/test_constraints.py (hunk 3 above)
Comment (katarinasupe) [reply tests/integration/test_constraints.py:173]: you're probably right

**Review by katarinasupe — CHANGES_REQUESTED** [review]

**Comment by katarinasupe** [comment]
I started the tests to see if everything works with your PR. I reviewed it, and you did catch nice typos, so thanks for that :D I will see if we can add your project/action to our PRs to avoid these things in the future, or to add a requirement to run it locally before the push. Let me know if you have some recommendations regarding that.

**Comment by jsoref** [comment]
We aren't currently using memgraph, but we're in the market for a new database. We're obviously more likely to select one with a vibrant developer community.

Once the PR is merged, in happy to make a PR for the workflow. The one I used is linked to in the initial PR description, however, I'll make some changes before I submit it as I don't encourage users to use prerelease software 😉.

I'm on vacation ATM, but I should be able to update PRs some time this week.

**Comment by katarinasupe** [comment]
Awesome to hear. I am open to any questions about Memgraph :) Discord is probably quicker, so looking forward to hopefully meeting you there, too 😄 

Enjoy your vacation, and I will hear from you soon 🌴

File: docs/reference/gqlalchemy/transformations/translators/nx_translator.md [hunk 6]
```diff
@@ -31,9 +31,9 @@ Generates Cypher queries for creating a graph by query groups.
 class NxTranslator(Translator)
 ```
 
-Uses original ids from Memgraph. Labels are encoded as properties. Since Networkx allows
+Uses original ids from Memgraph. Labels are encoded as properties. Since NetworkX allows
```
Comment (jsoref) [inline docs/reference/gqlalchemy/transformations/translators/nx_translator.md:34]: There are additional instances of me changing the case of `NetworkX`, e.g. here...

**Review by katarinasupe — APPROVED** [review]

**Comment by katarinasupe** [comment]
Hi @jsoref, I consider this PR done, and it will be added to the new GQLAlchemy release. Thank you for your contribution! 🙏

**Comment by katarinasupe** [comment]
Hi @jsoref, I accidentally added duplicate commits here after resolving conflicts. I wanted to rebase and remove the last six commits (duplicated) or maybe even all 10, but I did not have the necessary permissions. Can you give me the permissions or remove them? I just wanted to align with the current main branch since many of your changes involved the docs folder we changed in the last couple of weeks. Other solutions would be to create a new PR and run your action since our docs changed or to cherry-pick from this branch and create a new PR.

**Review by katarinasupe — APPROVED** [review]

**Pull request merged**